src/Microsoft.ML.StaticPipe/TextStaticExtensions.cs

```diff
@@ -571,11 +571,11 @@ public bool Equals(Reconciler other)
         /// Produces a bag of counts of ngrams (sequences of consecutive words of length 1-n) in a given tokenized text.
         /// It does so by hashing each ngram and using the hash value as the index in the bag.
         ///
-        /// <see cref="ToNgramsHash"/> is different from <see cref="WordHashBagEstimatorStaticExtensions.ToBagofHashedWords"/>
-        /// in a way that <see cref="ToNgramsHash"/> takes tokenized text as input while <see cref="WordHashBagEstimatorStaticExtensions.ToBagofHashedWords"/> tokenizes text internally.
+        /// <see cref="ApplyNgramHashing"/> is different from <see cref="WordHashBagEstimatorStaticExtensions.ToBagofHashedWords"/>
+        /// in a way that <see cref="ApplyNgramHashing"/> takes tokenized text as input while <see cref="WordHashBagEstimatorStaticExtensions.ToBagofHashedWords"/> tokenizes text internally.
         /// </summary>
         /// <param name="input">The column to apply to.</param>
-        /// <param name="hashBits">Number of bits to hash into. Must be between 1 and 30, inclusive.</param>
+        /// <param name="numberOfBits">Number of bits to hash into. Must be between 1 and 30, inclusive.</param>
```

Comment:
would be nice to align with @artidoro changes in his hashing scrubbing.

Reply:
I renamed `hashBits` to `numberOfHashBits` what's better?

Reply:
The `numberOfBits` in a hashing algorithm defaults to the number of output bits, I think.

Reply:
After thinking about it a bit, I think it might be better to name it `numberOfHashBits` everywhere. `numberOfBits` works well for `HashingTransformer`  which only does hashing, but sometimes, like in this case, there are settings that are relevant to hashing and some that are relevant to ngrams. I think it might be clearer for users to know that we are talking about the number of bits used for hashing.

Reply:
If so, we should rename maximumNumberOfIterations to maximumNumberOfTrainingIterations, ordered to orderedHashing, seed to hashSeed, right?

---
In reply to: [264369474](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264369474) [](ancestors = 264369474)